hoomd/mpcd/CellList.h

```diff
@@ -212,10 +212,10 @@ class PYBIND11_EXPORT CellList : public Compute
     void drawGridShift(uint64_t timestep);
 
     //! Gets the group of particles that is coupled to the MPCD solvent through the collision step
-    std::shared_ptr<ParticleGroup> getEmbeddedGroup() const
-        {
-        return m_embed_group;
-        }
+    // std::shared_ptr<ParticleGroup> getEmbeddedGroup() const
+    //     {
+    //     return m_embed_group;
+    //     }
```

Comment:
Why is this commented out? I assume it needs to be there.

Reply:
OK, then you can leave the method in and remove the commenting so no change is made.